packages/ui/src/kit/VSnackbar/VSnackbar.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div :class="cssClasses">
+    <div class="v-snackbar-container">
+      <svg-atom
+        v-if="icon"
+        class="v-snackbar-left-side"
+        :name="icon"
+      />
+      <div class="v-snackbar-content">
+        <slot />
+      </div>
+      <div
+        v-if="!isMultiLine"
+        class="v-snackbar-controls"
+      >
+        <slot name="control" />
+      </div>
+      <div
+        class="v-snackbar-right-side"
+        @click="onClose"
+      >
+        <div
+          class="v-snackbar-close-button"
+          @click="onClose"
+        >
+          <svg-atom name="close" />
+        </div>
+      </div>
+    </div>
+    <div
+      v-if="isMultiLine"
+      class="v-snackbar-controls"
+    >
+      <slot name="control" />
```

Comment:
We can have multiple items in control? If true - `controls`

Reply:
we have multiple items, but each item used as single slot. stories have this case as example
![image](https://user-images.githubusercontent.com/756731/77760363-445a9f00-7047-11ea-878d-241aae1b2391.png)

Reply:
I think - better have one slot for multiple elements

Reply:
hmm, how it will be looks?
```
<v-snackbar
              skin="warning"
              @close="onClose"
            >
              One line text with two actions
              <v-snackbar-link
                slot="controls"
                @click="onAction"
              >
                Action
              </v-snackbar-link>
              <v-snackbar-link
                slot="controls"
                @click="onAction"
              >
                Action
              </v-snackbar-link>
            </v-snackbar>
```

or
```
<v-snackbar
              skin="warning"
              @close="onClose"
            >
              One line text with two actions
              <template slot="controls">
                <v-snackbar-link
                  @click="onAction"
                >
                  Action
                </v-snackbar-link>
                <v-snackbar-link
                  @click="onAction"
                >
                  Action
                </v-snackbar-link>
              </template>
            </v-snackbar>
```